src/pageManager.js

```diff
@@ -705,12 +705,19 @@ export async function initialize() {
     // Register the pageManager content script for all URLs permitted by the extension manifest.
     const matchPatterns = permissions.getManifestOriginMatchPatterns();
 
-    // TODO register this content script so it may be unloaded later. 
+    const contentScriptId = "pageManager";
```

Comment:
could we use [getRegisteredContentScripts](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/scripting/getRegisteredContentScripts) here instead? use the filter feature and check for empty array?